packages/devextreme/js/common/ai-integration.d.ts

```diff
@@ -0,0 +1,140 @@
+/**
+ * @docid
+ * @namespace DevExpress.aiIntegration
+ * @public
+ */
+export interface Prompt {
+  /**
+   * @docid
+   * @public
+   */
+  system?: string;
+  /**
+   * @docid
+   * @public
+   */
+  user?: string;
+}
+
+/**
+ * @docid
+ * @namespace DevExpress.aiIntegration
+ * @public
+ */
+export interface RequestParams {
+  /**
+   * @docid
+   * @public
+   */
+  prompt: Prompt;
+  /**
+   * @docid
+   * @public
+   */
+  onChunk: (chunk: string) => void;
+}
+
+/**
+ * @docid
+ * @namespace DevExpress.aiIntegration
+ * @public
+ */
+export interface ResponseParams {
```

Comment:
How about just `Response` ? At 1st glance I wouldn't expect from a response obj to have params. Parameters are like input, but reponse is output.
```suggestion
export interface ResponseParams {
```

Reply:
I have no objection